bin/context-adapter

```diff
@@ -27,4 +27,4 @@
 
 var contextAdapter = require('../lib/context_adapter');
 
-contextAdapter.startup();
+contextAdapter.start(true);
```

Comment:
To avoid having to set it when running the tests or when running the CA in normal mode... :) It just avoids logging the proof-of-life message when running tests :) I preferred the option to enable or to disable it via code (see https://github.com/telefonicaid/context-adapter/blob/feature/enable-disable-proof-of-life-logging-reloaded/test/unit/context_adapter_test.js#L45)

Anyhow, I am removing the option to enable or disabling the proof-of-life logging and I am always enabling it since it adds an unneeded complexity to the code :) No big benefits :)